src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -21,7 +22,7 @@ public AuthenticationController(AuthenticationService authService,
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
+    public UserResponseDto register(@RequestBody @Valid UserRequestDto requestDto) {
```

Comment:
![image](https://user-images.githubusercontent.com/80981426/236183359-1d4d6d2c-c448-490a-8ea5-22d26d2745e9.png)
I don't see 2 and 5 points of the task, in your solution, make sure you do all of the above before sending it on review